pkg/pillar/base/kubevirt.go

```diff
@@ -5,12 +5,23 @@ package base
 
 import (
 	"os"
+	"regexp"
 	"strings"
+
+	uuid "github.com/satori/go.uuid"
 )
 
 const (
 	// EveVirtTypeFile contains the virtualization type, ie kvm, xen or kubevirt
 	EveVirtTypeFile = "/run/eve-hv-type"
+	// KubeAppNameMaxLen limits the length of the app name for Kubernetes.
+	// This also includes the appended UUID prefix.
+	KubeAppNameMaxLen = 32
+	// KubeAppNameUUIDPrefixLen : length of the app UUID prefix appended to the app
```

Comment:
If it is appended shouldn't it be called a SuffixLen?

Reply:
Well, it is UUID prefix (we take first few characters of the app UUID), which is then used as the pod name suffix. So hard to say if prefix or suffix is better name :)

Reply:
Ah yes, it should be called the KubeAppNameSuffixFromUUIDPrefixLen. Just kidding.
I don't know if others will be thrown off by the current name+comment. Maybe the comment can say something like "length of the initial characters from the app UUID which are appended to the app name"?